src/shims/intrinsics.rs

```diff
@@ -137,48 +133,51 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
                 // even if the type they wrap would be less aligned (e.g. AtomicU64 on 32bit must
                 // be 8-aligned).
                 let align = Align::from_bytes(place.layout.size.bytes()).unwrap();
-                this.memory.check_ptr_access(place.ptr, place.layout.size, align)?;
+                this.memory
+                    .check_ptr_access(place.ptr, place.layout.size, align)?;
 
                 // binary_op will bail if either of them is not a scalar
-                let eq = this.overflowing_binary_op(mir::BinOp::Eq, old, expect_old)?.0;
+                let eq = this
+                    .overflowing_binary_op(mir::BinOp::Eq, old, expect_old)?
+                    .0;
                 let res = Immediate::ScalarPair(old.to_scalar_or_undef(), eq.into());
                 this.write_immediate(res, dest)?; // old value is returned
-                // update ptr depending on comparison
+                                                  // update ptr depending on comparison
                 if eq.to_bool()? {
                     this.write_scalar(new, place.into())?;
                 }
             }
 
-            "atomic_or" |
-            "atomic_or_acq" |
-            "atomic_or_rel" |
-            "atomic_or_acqrel" |
-            "atomic_or_relaxed" |
-            "atomic_xor" |
-            "atomic_xor_acq" |
-            "atomic_xor_rel" |
-            "atomic_xor_acqrel" |
-            "atomic_xor_relaxed" |
-            "atomic_and" |
-            "atomic_and_acq" |
-            "atomic_and_rel" |
-            "atomic_and_acqrel" |
-            "atomic_and_relaxed" |
-            "atomic_nand" |
-            "atomic_nand_acq" |
-            "atomic_nand_rel" |
-            "atomic_nand_acqrel" |
-            "atomic_nand_relaxed" |
-            "atomic_xadd" |
-            "atomic_xadd_acq" |
-            "atomic_xadd_rel" |
-            "atomic_xadd_acqrel" |
-            "atomic_xadd_relaxed" |
-            "atomic_xsub" |
-            "atomic_xsub_acq" |
-            "atomic_xsub_rel" |
-            "atomic_xsub_acqrel" |
-            "atomic_xsub_relaxed" => {
+            "atomic_or"
+            | "atomic_or_acq"
```

Comment:
This is bad, they should all be left-aligned.

Reply:
I am not sure what you mean by "left-aligned"

Reply:
oh, maybe I do. See the new style

Reply:
The first `"` should be in the same column everywhere.